pallets/runtime/tests/src/fee_details.rs

```diff
@@ -54,26 +54,24 @@ fn cdd_checks() {
                 InvalidTransaction::Custom(TransactionError::InvalidAuthorization as u8)
             );
 
-            // call to accept being a multisig signer should fail when authorizer does not have a valid cdd (expired)
             create_multisig_default_perms(
                 alice_account.clone(),
                 create_signers(vec![alice_account.clone()]),
                 1,
             );
```

Comment:
We shouldn't have been creating this MS with the caller also being a signer.  Since it shouldn't be possible for the caller `alice` to become a signer on the MS (unless they also leave their DID before accepting the auth).  Also it makes the tests below very confusing.  Can you use `charlie` or another account as the caller here.  That will make at least some of the tests below to return `charlie` as the payer.

Since we want to ensure that the returned payer from `get_valid_payer` should be the primary key of the DID that created the authorization.